src/documentation/programming-guide.md

```diff
@@ -1072,7 +1072,7 @@ If your input `PCollection` uses the default global windowing, the default
 behavior is to return a `PCollection` containing one item. That item's value
 comes from the accumulator in the combine function that you specified when
 applying `Combine`. For example, the Beam provided sum combine function returns
-a zero value (the sum of an empty input), while the max combine function returns
+a zero value (the sum of an empty input), while the min combine function returns
```

Comment:
`min` returns a maximal `value`?

Don't you want to leave this at `max`?

Reply:
It should be `min`, because the identity of `min` is `maximal value`.